CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Tracking/AnalyticsManager.swift

```diff
@@ -136,9 +121,22 @@ public class AnalyticsManager {
     private static func processEventsQueue() {
```

Comment:
[suggestion] I think we should rename this method. What do you think about `trackEventsFromQueue` or `trackQueueEvents`?

Reply:
@ValentinaIancu-Gini if we will move out tracking events code from this method and I suggest to choose something from this:
`releaseEventsQueue`
`processEventsQueue`
`flushEventsQueue`

Reply:
mhh let's keep it as it is`processEventsQueue` but maybe we can add a comment to this method and describe what is actually doing.